iup/eval_projections.py

```diff
@@ -1,5 +1,100 @@
+import polars as pl
 import altair as alt
 
+# get mspe #
+def get_mspe(
+        data_df,
+        pred_df,
+        var
+):
+    """
+    Calculate the mean squared prediction error between the observed
+    data and prediction when both are available.
+
+    Parameters:
+    --------------
+    data_df: polars.DataFrame
```

Comment:
Perhaps we should insist that `data_df` and `pred_df` are IncidentUptakeData polars dataframes, specifically.